start.sh

```diff
@@ -0,0 +1,30 @@
+#!/bin/bash
+#
+# Start script for document-generator
+
+APP_DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd )"
+
+if [[ -z "${MESOS_SLAVE_PID}" ]]; then
+    source ~/.chs_env/private_env
+    source ~/.chs_env/global_env
+    source ~/.chs_env/document-generator/env
+
+    PORT="${DOCUMENT_GENERATION_SERVICE_PORT:=4017}"
```

Comment:
We shouldn't be assigning the `4017 port. Because when we check in the code if that PORT is null it should throw an error not start on a random port.